es-vue-base/src/lib-components/EsAccordion.vue

```diff
@@ -1,18 +1,24 @@
 <template>
     <div class="EsAccordion border-bottom border-light rounded-bottom">
-        <header role="tab">
-            <es-button
-                class="EsAccordion-button h-auto align-items-center d-flex font-weight-bold justify-content-between px-100 px-sm-200 py-100 rounded-0 text-body text-left text-decoration-none w-100"
+        <header
+            role="tab"
+            class="position-relative d-inline-block w-100">
```

Comment:
Would removing the `d-inline-block w-100` still work, as I believe the `<header>` tag would render at its default `display: block` and go the full width? If so, could be simpler.